### ISSUE TITLE
compute and display slope of speedup plots

### ISSUE THREAD

**Issue by prudhomm** [feelpp/benchmarking#61]
we should compute the slopes of speedup using eg using linear regression, and display them in the legend next to the label

**Comment by tanguypierre** [comment]
Linear regression computed with _sklearn.linear_model_
Exact values, regression values and slopes are now displayed in a table after the graph, with a precision from 3 decimals

**Comment by tanguypierre** [comment]
Addition of an _on/off button_ for displaying the regression line or not. (same color, but thinner and dashed line than real curve).
When activated, the dashed line appears in the legend showing the slope with a precision from 1 decimal.

_**Note**: Complete interactivity couldn't have been reached_ 
If a real curve is deselected, its regression line will appear by pushing the button

**Comment by JavierCladellas** [comment]
Done in #53
